index.js

```diff
@@ -232,11 +225,42 @@ function linux (args, sudo, binary) {
         sink(args, proc.pid, folder, clock)
       )
     })
+  })
 
+  proc = spawn('sudo', [
```

Comment:
Covered in it though is the change of the variable declarations that moved to the top.